### PR TITLE
Added SSR1XXX devices, 1515B, and added speed to the map

### PR DESCRIPTION
**Pull request by BenMatase**
Also fixed Silicom sku names to be consistent with dmidecode output

### CODE REVIEW

File: devicemaps.json [hunk 1]
```diff
@@ -704,6 +832,1363 @@
...
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "lte-0",
+                "description": "LTE network interface"
+              }
+            }
+          }
+        ]
+      },
+      "SSR130-AE-TAA": {
```
Comment (salegria8) [inline devicemaps.json:1280]: is there a list somewhere with all the different skus? I'm looking through this json but want to make sure I dont miss any. I'm looking to add more unit tests in https://github.com/mistsys/mist-agent-go/pull/655

Comment (BenMatase) [reply devicemaps.json:1280]: Yea the keys in this map are definitive. There is also a list on the mist side

**Comment by cmain** [comment]
Do we have a task to update the template globals that ship with SSR to include these changes?

cc @reidstidolph

File: devicemaps.json [hunk 2]
```diff
@@ -704,6 +832,1363 @@
...
+      },
+      "SSR1200": {
+        "clei": "INMJV10HRA",
+        "part_number": "650-147980",
+        "ethernet": [
+          {
+            "pciAddress": "0000:03:00.0",
+            "type": "WAN",
+            "speed": 1000,
+            "name": "ge-0-0-0",
```
Comment (avijitvjuniper) [inline devicemaps.json:1536]: This is going to be "mgmt-0-0-0"

**Review by avijitvjuniper — DISMISSED** [review]
@reidstidolph and I will be posting the changes to the PR to incorporate changes to the SSR1200 and removal of the SSR1500 for now. SSR1500 spec will be finalized shortly and so is being omitted from this Pull Request

**Comment by agrawalkaushik** [comment]
> @reidstidolph and I will be posting the changes to the PR to incorporate changes to the SSR1200 and removal of the SSR1500 for now. SSR1500 spec will be finalized shortly and so is being omitted from this Pull Request

This PR is to get this repo up-to-date with what we have in our i95 source code. You should open up your own PR to update the SSR1200 and SSR1500 stuff. Just let us know when you merge it so we can fetch the changes for building an image.

File: devicemaps.json [hunk 3]
```diff
@@ -704,6 +832,1363 @@
...
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "lte-0",
+                "description": "LTE network interface"
+              }
+            }
+          }
+        ]
+      },
+      "SSR1200": {
```
Comment (agrawalkaushik) [inline devicemaps.json:1528]: @BenMatase for now lets just remove all the SSR1xxx and let Avi and Reid provide the updates there.

File: devicemaps.json [hunk 4]
```diff
@@ -704,6 +832,1363 @@
...
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "lte-0",
+                "description": "LTE network interface"
+              }
+            }
+          }
+        ]
+      },
+      "SSR130-AA": {
```
Comment (agrawalkaushik) [inline devicemaps.json:1404]: uff - we desperately need to implement aliasing that we discussed @BenMatase 😄

**Review by agrawalkaushik — APPROVED** [review]

**Comment by agrawalkaushik** [comment]
> Do we have a task to update the template globals that ship with SSR to include these changes?
> 
> cc @reidstidolph

@cmain it's a work in progress but the overall idea is this:

* This repo will maintain the master list. We will be adding more schema validation and CI/CD job to validate the schema etc (in the coming weeks)
* our 128T repo will source the data from this GH repo - we can control what we pull in via git tags

The idea is for PLM to maintain this list and engineering to just source it blindly. If the same applies to the templates than this might be a good place to store them long term.

**Comment by reidstidolph** [comment]
> 

Roger that. Will submit changes in a separate PR.

**Review by reidstidolph — APPROVED** [review]

**Pull request merged**